ios/pillarwallet.xcodeproj/project.pbxproj

```diff
@@ -524,7 +524,7 @@
 				CODE_SIGN_ENTITLEMENTS = pillarwallet/pillarwallet.entitlements;
 				CODE_SIGN_STYLE = Manual;
 				CURRENT_PROJECT_VERSION = 1;
-				DEAD_CODE_STRIPPING = NO;
+				DEAD_CODE_STRIPPING = YES;
```

Comment:
What was the reason to change this?

Reply:
I couldn't compile iOS without that. I was getting errors like:
`Undefined symbol: _swift_getFunctionReplacement`
`Undefined symbol: _swift_getOrigOfReplaceable`
I'll remove native iOS changes from this PR, since I added them by accident anyway and it works for others, maybe it's just my xcode installation.

Reply:
Thanks 👍